lib/src/annotations/constants.dart

```diff
@@ -0,0 +1,159 @@
+/// Available Annotations
+enum AnnotationTypes {
```

Comment:
Change the plural to singular , `AnnotationTypes` -> `AnnotationType` for all the classes